library/erlang

```diff
@@ -1,21 +1,40 @@
-# maintainer: denc716@gmail.com (@c0b)
+# this file is generated via https://github.com/c0b/docker-erlang-otp/blob/ea32d5f6f1735f9f55bee04b112166da96eb9c73/generate-stackbrew-library.sh
 
-18.3.3: git://github.com/c0b/docker-erlang-otp@2daeab9ef204095169b3cc96def31ce08a5f7358 18
-18.3:   git://github.com/c0b/docker-erlang-otp@2daeab9ef204095169b3cc96def31ce08a5f7358 18
-18:     git://github.com/c0b/docker-erlang-otp@2daeab9ef204095169b3cc96def31ce08a5f7358 18
-latest: git://github.com/c0b/docker-erlang-otp@2daeab9ef204095169b3cc96def31ce08a5f7358 18
+Maintainers: denc716@gmail.com (@c0b)
```

Comment:
Would you mind updating this to be `Name <email> (@github)`? (ie, `Mr C0B <denc716@gmail.com> (@c0b)` is sufficient IMO if you'd like)